nsxt/resource_nsxt_policy_vlan_segment_test.go

```diff
@@ -201,8 +201,15 @@ func testAccNsxtPolicyVlanSegmentExists(resourceName string) resource.TestCheckF
 		if resourceID == "" {
 			return fmt.Errorf("Policy VLAN Segment resource ID not set in resources")
 		}
+		var err error
```

Comment:
Shall we use resourceNsxtPolicySegmentExists here?

Reply:
Done